### DISCUSSION TITLE
I installed Terraria 1.4.3.2 sucsessfully but its for windows

### DISCUSSION THREAD

**Discussion by woodenlalaloo** [SteamRE/DepotDownloader#324] [Q&A] [answered]
The app was installed just fine but even when specifying macOS (dotnet DepotDownloader.dll -app 105600 -depot 105601 -manifest 4362065484837434637 -username woodenlalaloo -password lakehouse115$ -os macos) it still downloads an exe and .bat files for the windows version.

**Comment by psychonic** [comment, marked as answer]
You specified `-depot` as `105601`, which is the Windows depot, negating the `-os` option. It looks like the Mac depot is 105603 (https://steamdb.info/app/105600/depots/). Either specify that instead, or remove the `-depot` specification altogether.

I believe you can also the `-os` specification as well. You mentioned in the other thread that you are on a Mac, so it should default to pulling content available on that platform.

**Reply by woodenlalaloo** [reply]
ah ok, i will try that
